.github/workflows/api-docker.yml

```diff
@@ -5,7 +5,7 @@ on:
     paths:
```

Comment:
can you check `mkdocs serve` and be sure this PR hits 90% test coverage before you resubmit?

Reply:
There were a few pre-existing errors that seem to have come from the marketing review prs. Everything should be taken care of now.